config/software/ruby.rb

```diff
@@ -163,6 +167,7 @@
   configure_command = ["--with-out-ext=dbm,readline",
                        "--enable-shared",
                        "--disable-install-doc",
+                       "--with-jemalloc",
```

Comment:
consider making this a conditional add to `configure_command` based on the decision made above about `dependency "jemalloc"`

Reply:
@jmartin-r7 Good spot; I've added added this now